### PR TITLE
New packages: vit-2.1.0 and python3-tasklib-2.4.3

### PR DESCRIPTION
**Pull request by Barbarossa93**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/python3-tasklib/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-tasklib'
+pkgname=python3-tasklib
+version=2.4.3
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-wheel python3-pip task"
```
Comment (paper42) [inline srcpkgs/python3-tasklib/template:7]:
```suggestion
hostmakedepends="python3-setuptools task"
```

File: srcpkgs/python3-tasklib/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+depends="python3 task"
```
Comment (paper42) [inline srcpkgs/python3-tasklib/template:8]: pytz and python3-tzlocal are missing

File: srcpkgs/python3-tasklib/template (hunk 1 above)
Comment (Barbarossa93) [reply srcpkgs/python3-tasklib/template]: Thanks, for catching that

Comment (Barbarossa93) [reply srcpkgs/python3-tasklib/template]: Hmm, the checks fail without those. Are you sure we don't need them?

File: srcpkgs/python3-tasklib/template (hunk 2 above)
Comment (Barbarossa93) [reply srcpkgs/python3-tasklib/template]: Is there info on the PyPi site that gives dependencies? All I could gather was that taskwarrior and python are needed ([src](https://pypi.org/project/tasklib/))

File: srcpkgs/python3-tasklib/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/python3-tasklib/template]: put them to checkdepends and also add pytest

Comment (paper42) [reply srcpkgs/python3-tasklib/template]: I meant to add the dependencies there, not pip

File: srcpkgs/python3-tasklib/template (hunk 2 above)
Comment (paper42) [reply srcpkgs/python3-tasklib/template]: Usually setup.py, setup.cfg or pytest.ini contains install_requires or an equivalent, also checks will fail if you forget to specify a dependency and don't add pip to checkdepends (which defeats the purpose of having checks).
https://github.com/GothenburgBitFactory/tasklib/blob/f7672ddebe6d4f4ff8fdb743df0b2939ba66ec78/setup.py#L25

File: srcpkgs/python3-tasklib/template (hunk 1 above)
Comment (Barbarossa93) [reply srcpkgs/python3-tasklib/template]: All host dependencies?

File: srcpkgs/python3-tasklib/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 task python3-tzlocal python3-pytz"
+checkdepends="task"
+short_desc="Library for interacting with taskwarrior databases (Python3)"
+maintainer="Cullen Ross <cullenrss@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/robgolding/tasklib"
```
Comment (ahesford) [inline srcpkgs/python3-tasklib/template:13]: This seems to redirect to https://github.com/robgolding/tasklib for me.

Comment (Barbarossa93) [reply srcpkgs/python3-tasklib/template]: Changed the link the the Gothenburg one. Great catch!

**Comment by Barbarossa93** [comment]
Unless I placed the required env vars in the wrong place, this PR is all set.

File: srcpkgs/vit/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'vit'
+pkgname=vit
+version=2.1.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3 task python3-pytz python3-urwid python3-tzlocal python3-tasklib"
+checkdepends="python3-devel task python3-tzlocal python3-urwid python3-tasklib"
```
Comment (paper42) [inline srcpkgs/vit/template:8]:
```suggestion
depends="task python3-pytz python3-urwid python3-tzlocal python3-tasklib"
checkdepends="python3-pytest $depends"
```

File: srcpkgs/vit/template [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+maintainer="Cullen Ross <cullenrss@gmail.com>"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+distfiles="${PYPI_SITE}/v/vit/vit-${version}.tar.gz"
+checksum=fd34f0b827953dfdecdc39f8416d41c50c24576c33a512a047a71c1263eb3e0f
+
+pre_check() {
+	export TERM=linux
+	export TERMINFO=/usr/share/terminfo
+}
```
Comment (paper42) [inline srcpkgs/vit/template:19]:
```suggestion
do_check() {
	TERM=linux python3 -m pytest
}
```

Comment (Barbarossa93) [reply srcpkgs/vit/template]: Thanks! I wasn't sure what the default do_check() function was, so this change makes waaaay more sense.

**Pull request merged**